### ISSUE TITLE
unused dependencies in cpanfile?

### ISSUE THREAD

**Issue by lelutin** [oetiker/SmokePing#334]
Hi there @oetiker,

I'm currently reviewing dependencies for the debian package of smokeping as I'm preparing to send version 2.8.2.

I saw two lines in the `cpanfile` at the top of this repository for which I could not find any reference/inclusion in the codebase, so I'm wondering if they are still necessary or if they can be taken out:

 * JSON::MaybeXS
 * Object::Result

Cheers!

**Comment by oetiker** [comment]
Yes, please :) if things break, the tests should show

**Comment by lelutin** [comment]
digging into the commit history, I can see that both of the lines were added in d38a064e0f0f57f5dc8a947a6a28d64cb710abad for InfluxDB. so maybe this issue is actually not relevant?

**Comment by oetiker** [comment]
Indeed: https://metacpan.org/release/RAPHI/InfluxDB-HTTP-0.04/source/lib/InfluxDB/HTTP.pm and they are not listed as dependencies in Makefile.PL ... yey

**Comment by oetiker** [comment]
you could add a comment to the cpanfile :)

**Comment by lelutin** [comment]
oh wow good find. I had checked dependencies on metacpan.org and was a bit puzzled there.

ok good, I'll add a comment in the cpanfile then so we'll remember about this. and it sounds like maybe a bug report on that lib is in order too..
